### ISSUE TITLE
Game: Store game results

### ISSUE THREAD

**Issue by nicokaiser** [nicokaiser/bandoneon#107]
Store results (in localStorage) and make result shareable somehow.

**Comment by nicokaiser** [comment]
Yes, I still develop it. Occasionally I update the libraries and fix bugs. Did not have much time to implement new features though...

**Comment by marcelomazza** [comment]
Nice! I use it daily, I'll try to collaborate with some. It's a great app, congrats.
